Resources/views/CRUD/edit_orm_one_association_script.html.twig

```diff
@@ -47,7 +47,12 @@ This code manage the one-to-many association field popup
                     return;
                 }
 
-                jQuery('#field_container_{{ id }}').replaceWith(html); // replace the html
+                if (jQuery('#field_widget_{{ id }}').children().length) {
+                    var $newRow = jQuery(jQuery.parseHTML(html)).find('tr:last');
```

Comment:
@auipga The code does submit the whole form to the server in order to rebuild the collection field.
So the real issue here is select2 not synchronizing changes to the underlying input, because the submit event is stopped before select2 has a chance to get at it.

So your first move would be to try to remove the `event.stopPropagation()` call. I don't think it's needed anyway.
If you're lucky then great !
If not, then you need to find a way to force select2 to update the underlying input _before_ the ajax call.
Something like this (not tested but should give you the idea):
```js
// panzer division method, so this might even work with other plugins
$('#field_container_{{ id }}').find(':input').change();
```